### DISCUSSION TITLE
Onyx Boox Darwin Android 4.2

### DISCUSSION THREAD

**Discussion by utmuk** [plateaukao/einkbro#152] [Q&A]
Is there a way to make this awesome browser work on Android 4.2? I have a e-ink device Onyx Boox Darwin C67ML, which can't be upgraded to Android 5 :(

**Comment by plateaukao** [comment]
No, Android 4.2 is a system released more than 10 years ago. It takes too much effort to make current EinkBro work on it, and do the maintanence. Even if EinkBro could run on it, it would be very slow and lack of lots of features due to OS system support is bad.

**Comment by poVoq** [comment]
It also does't work on Onyx Boox Note, which is on Android 6 and still supported with updates by the vendor.

**Reply by plateaukao** [reply]
It's great to know that BOOX Note is still supported with updates by Onyx. :)

**Reply by poVoq** [reply]
Well, the last system update was about six months ago, but I guess that counts as supported for a device that old.
